core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -258,7 +258,7 @@ trait WorkspaceComponent {
       loadWorkspaces(workspaceQuery)
 
     def listWithBillingProject(billingProject: RawlsBillingProjectName): ReadAction[Seq[Workspace]] =
-      workspaceQuery.withBillingProject(billingProject).read
+      workspaceQuery.withBillingProject(billingProject).withVersion(WorkspaceVersions.V2).read
```

Comment:
The filtering was being done inline where this was being called, so it seemed easier to just put this here instead.